CHANGELOG.md

```diff
@@ -2,6 +2,113 @@
 
 All notable changes to this project will be documented in this file.
 
+## Unreleased - as of Sprint 45 end 2016-08-22
+
+### [Added](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+45+Ending+Aug+22%2C+2016%22+label%3Aenhancement)
+
+- Automate
+  - Enhanced messaging for provisioning: Displayed elements
+    - ManageIQ Server name
+    - Name of VM/Service being provisioned
+    - Current Automate state machine step
+    - Status message
+    - Provision Task Message
+    - Retry count (when applicable)
+  - New method `taggable?` to programmatically determine if a Service Model class or instance is taggable.
+  - Generic Objects: Model updates
+    - Associations
+    - Tagging
+    - Service Methods: `add_to_service / remove_from_service`
+  - Git Automate support
+    - Branch/Tag support
+    - Contents are locked,and can be copied to other domains for editing
+    - Editable properties
+      - Enabled/Disabled
+      - Priority
+      - Removal of Domain
+    - Dedicated Server Role to store the repository
+- Platform
+  - PostgreSQL High Availability
+    - Primary/Standby DB config in Appliance Console
+    - Database-only appliance config in Appliance Console
+    - Failover Monitor
+  - Tenancy
+    - Groundwork in preparation for supporting multiple entitlements
+    - ApplicationHelper#role_allows and User#role_allows? combined and moved to Rbac
+    - Post refresh hook to queue mapping of Cloud Tenants
+  - Database maintenance scripts added to appliance
+- Providers
+  - Containers: Models for container deployments
+  - Google
```

Comment:
@mfeifer according to @selmanj, we can say "Google Compute Engine" since everything we're accessing today is under the "Compute" umbrella.

Later, when there's more functionality under other services, we can expand to "Google Cloud Platform".